snap/pack/pack.go

```diff
@@ -211,24 +211,31 @@ func copyToBuildDir(sourceDir, buildDir string) error {
 	})
 }
 
-func prepare(sourceDir, targetDir, buildDir string) (snapName string, err error) {
+// LoadValidate attempts to load and validate  snap data from source directory
+func LoadValidate(sourceDir string) (*snap.Info, error) {
```

Comment:
I'm very bad with names, but this is a bad name :-)

Reply:
Hmm I could rename it to `func CheckSkeleton(sourceDir string) error` and bury the ugly name in the package :)